### DISCUSSION TITLE
Which pin woke it up, esp32.wake_on_ext1()?

### DISCUSSION THREAD

**Discussion by davefes** [micropython/micropython#10929] [ESP32] [answered]
I got this far"

```py
sensor1 = Pin(26, mode = Pin.IN)
sensor2 = Pin(27, mode = Pin.IN)

esp32.wake_on_ext1(pins = (sensor1, sensor2), level = esp32.WAKEUP_ANY_HIGH)
```
now I would like to know which pin woke it up.  Is this possible in software?

**Comment by karfas** [comment, marked as answer]
You will need to patch and rebuild.
PR #7990 is not yet merged.

**Reply by davefes** [reply]
Thanks for that.

**Comment by sosi-deadeye** [comment]
Thanks, that you're asking this question because I need this too.
